client/src/App.js

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from "react";
+import Jumbotron from "./components/Jumbotron";
+import Nav from "./components/Nav";
+import Input from "./components/Input";
+import Button from "./components/Button";
+import API from "./utils/API";
+import { RecipeList, RecipeListItem } from "./components/RecipeList";
+import { Container, Row, Col } from "./components/Grid";
+
+import axios from "axios";
+
+class App extends Component {
+  state = {
+    articles: [],
+    articleSearch: "california",
+    isLoading: true,
+  };
+
+  getArticles() {
+        // This variable will be pre-programmed with our authentication key
+        // (the one we received when we registered)
+        var authKey = "b9f91d369ff59547cd47b931d8cbc56b:0:74623931";
+
+        // queryURLBase is the start of our API endpoint. The searchTerm will be appended to this when
+        // the user hits the search button
+        var url = "https://api.nytimes.com/svc/search/v2/articlesearch.json?api-key=" +
+          authKey + "&q="+ this.state.articleSearch;
```

Comment:
destructuring state, using string interpolation, and using let would be better practices to use here.